dandi/upload.py

```diff
@@ -335,7 +332,7 @@ def upload_agg(*ignored: Any) -> str:
 
     if sync:
         relpaths: List[str] = []
-        for p in original_paths:
+        for p in paths:
```

Comment:
I wonder if there was some good reason to use `original_paths` here instead of `.absolute()`ified ones... can't think of anything and (to be removed) comment on expanding and "validating" does not help (was there some which was removed? or did we use normpath not abspath at some point?)

Reply:
I see, so they were overloaded with the ones from `find*files` which is no longer the case iirc, so not needed indeed.  I don't think `absolute` should have negative effect there.. let's proceed. thanks!